README.md

```diff
@@ -172,6 +173,16 @@ Be aware of possible security threats and problems. You should at least be famil
 
 Good generic security guidelines would be:
 
+## Docker
+
+**Using Docker will not make your service more secure.** Generally, you should consider at least following things if using Docker:
+
+- Don't run any untrusted binaries inside Docker containers
+- Create unprivileged users inside Docker containers and run binaries using unprivileged user instead of root whenever possible
+- Periodically update the dependencies inside Docker containers
+- Periodically update your Docker hosts
+- Multiple containers running on same host will have some level of access to other containers and the host itself. Properly secure all containers and hosts
```

Comment:
Could we be a bit more specific about securing? For example, containers that don't need the network can run without any network access, capabilities can be removed etc.

Reply:
We definitely could and should be more specific. However, this is the part where my knowledge ends :(

Reply:
I'll contribute some words to this PR about that :)